### ISSUE TITLE
Create separate OpenAPI specs for different packages.

### ISSUE THREAD

**Issue by kyngs** [smallrye/smallrye-open-api#1669]
(I'm a newbie and I'm not sure whether this is possible, or whether this is a request for this repository, or https://github.com/eclipse/microprofile-open-api/)

I have the following folder structure for my resources:
```
api/
├─ v0/
│  ├─ Example V0 Resource
├─ v1/
│  ├─ Example V1 Resource
```

Currently, the V0 a V1 resources would get put into the same openapi spec. I believe it would be optimal to have a separate spec for the v0 package and a separate spec for the v1 package.

Is it possible to do this? If not, why not?

 *Sorry for my ignorance if I'm missing something here*

**Comment by kyngs** [comment]
> The maven plugin provides the scanProfiles and scanExcludeProfiles options. These can be used to selectivly include classes for one execution of the plugin.
> 
> Was implemented in #967.

So there's no possible way for me to do this with Gradle?

**Comment by kyngs** [comment]
> @kyngs I had a look around, and the properties are at least present in the config class for the gradle plugin. https://github.com/smallrye/smallrye-open-api/blob/main/tools/gradle-plugin/src/main/java/io/smallrye/openapi/gradleplugin/Configs.java#L61
> 
> No idea how to configure it atm tough, sorry - never worked with gradle myself.

Pheew, I'm using the quarkus-smallrye-openapi for Quarkus, so I suppose I should go ask there. Thanks for your help tho.
